protocols/banklessvault.eth/index.json

```diff
@@ -0,0 +1,32 @@
+{ 
+	"cname": "banklessvault.eth",
```

Comment:
for consistency, consider `bankless` or `banklessdao` maybe for cname instead of the ENS name (with a dot)

Reply:
Yeah I think `.eth` in the `cname` will actually break the harvester as is.